docs/organizer-guide/meetups/starting.rst

```diff
@@ -1,11 +1,10 @@
 Starting a Meetup
 =================
 
-Interested in fostering a lovely gathering of documentarians in your neck of the woods?
-This is the place to start!
+If you've been to a Write The Docs conference, you may have the need to "continue the conversation".
+With a little help, you can make it happen with a Meetup in your area!
```

Comment:
I'm going to think about this... "continue the conversation" has been my personal tagline for "Why WTD"